compat/mingw.c

```diff
@@ -3541,7 +3557,14 @@ int is_path_owned_by_current_sid(const char *path)
 			 * okay, too.
 			 */
 			result = 1;
-		else if (git_env_bool("GIT_TEST_DEBUG_UNSAFE_DIRECTORIES", 0)) {
+		else if (IsWellKnownSid(sid, WinWorldSid) &&
+			 git_env_bool("GIT_TEST_DEBUG_UNSAFE_DIRECTORIES", 0) &&
+			 !acls_supported(path)) {
+			/*
+			 * On FAT32 volumes, ownership is not actually recorded.
+			 */
+			warning("'%s' is on a file system that cannot represent ownership", path);
```

Comment:
@derrickstolee do you think "that does not record ownership" would be more accurate/elegant?

Reply:
I agree that "does not record" is the most accurate. I read "cannot represent" as "cannot tell me" which is different than "doesn't even keep track of".